### PR TITLE
Added options for content input, removing separator and margins

### PR DESCRIPTION
**Pull request by xavijam**
CR @piensaenpixel

### CODE REVIEW

File: src/scss/cdb-components/forms/option_input.scss [hunk 1]
```diff
@@ -129,6 +132,9 @@
     }
   }
 }
+.CDB-OptionInput-item--noSeparator {
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/option_input.scss]: i suppose you want the empty content in the pseudoelement, right? ::after

**Comment by piensaenpixel** [comment]
👍

File: src/scss/cdb-components/forms/option_input.scss (hunk 1 above)
Comment (xavijam) [inline src/scss/cdb-components/forms/option_input.scss]: Yep man, thanks!

**Comment by xavijam** [comment]
Renamed file with hyphens instead of underscores, ok?

**Pull request merged**